reviews/review-02+2020-11-11.md

```diff
@@ -0,0 +1,77 @@
+## What do you think is good about this manuscript?
+
+The range of topics covered is thorough without being overwhelming.
+There is a tendency to attempt to cram every single aspect of software
+engineering into a book like this without acknowledging feasible
+limits. Yes, it is over 450 pages in it's current draft, but that
+seems a reasonable amount given the intended audience.
+
+## What reservations do you have about this manuscript?
+
+There is a sense in which the intended audience may not view
+themselves as software engineers (i.e., they are likely researchers in
+some other specific scientific discipline). As such, the term
+"research software engineering" may miss the mark. That said, there
+are plenty of people who do use this term and there are not really any
+effective alternatives in common use. So the title is still likely to
+reach it's intended audience.
+
+## In terms of content and structure, do you think the chapters need any revisions, such as additions or omissions?
+
+I'm impressed at the depth and breadth of the content, so I see no
+need for major revisions beyond normal copy-editing, finishing the
+figures, etc.
+
+## Are the chapter titles/subtitles appropriate?
+
+The chapter titles form an appropriate sequence. I might consider
+swapping Chapters 12 (Provenance) and 13 (Python Packaging), but the
+remaining topics build in a sensible order.
+
```

Comment:
My preference would be to leave the order of the Provenance and Packaging chapters as is. The main reason for this is that if we do provenance after packaging, it kind of implies that the code associated with a research paper can't be made available unless it is packaged for a general audience, which is definitely not the signal we want to send.

Reply:
I have no strong opinion - if no one else strongly prefers a change, let's leave it as is.